aspnet/mvc/views/forms/sample/final/Startup.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace FormsTH
+{
+    public class Startup
+    {
+         public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            app.UseDeveloperExceptionPage();  
```

Comment:
Don't demonstrate unconditional use of this feature. Should not be used in production.